Discovery/src/main/resources/application.yml

```diff
@@ -1,5 +1,5 @@
 server:
-  port: 9003
+  port: 8761
```

Comment:
What is the reason to change the port

Reply:
8761 is default port for discovery so I changed it to the default port